static/index.json

```diff
@@ -1,32 +1,37 @@
 [
+    {
```

Comment:
Could you also removes lines 2-6. This is what I was referring to the in the updates in the README PR.